src/components/Header.tsx

```diff
@@ -21,9 +22,11 @@ type HeaderProps = {
 
     /** Additional header container styles */
     containerStyles?: StyleProp<ViewStyle>;
+
+    imageHrefLink?: string;
```

Comment:
This is a common component, so it shouldn’t include specific names in its properties. Also, please add a description for this prop.
```suggestion
    subTitleLink?: string;
```